src/routes/_api/utils.js

```diff
@@ -10,6 +10,10 @@ export function basename (instanceName) {
 }
 
 export function auth (accessToken) {
+  if (accessToken === undefined || accessToken === null) {
+    return {}
+  }
+
```

Comment:
I would rather we fix the places where the accessToken is null. Usually I'm pretty specific about not including the accessToken in cases where it's not needed (e.g. `/api/v1/instance`).

Reply:
The accessToken may be needed to view toots that have privacy settings (e.g. DMs, follower-only).

I am not sure if it is *really* needed for viewing account profiles though. I think not, but maybe it flows through to the timeline which would then need it.